target/linux/ath79/image/generic.mk

```diff
@@ -805,6 +820,27 @@ define Device/engenius_ecb1750
 endef
 TARGET_DEVICES += engenius_ecb1750
 
+define Device/engenius_enh202-v1
+  SOC := ar7240
+  DEVICE_VENDOR := EnGenius
+  DEVICE_MODEL := ENH202
+  DEVICE_VARIANT := v1
+  DEVICE_PACKAGES := rssileds
+  IMAGE_SIZE := 4864k
+  KERNEL := kernel-bin | append-dtb | lzma | uImage lzma -M 0x4f4b4c49
+  LOADER_TYPE := bin
+  LOADER_FLASH_OFFS := 0x1b0000
+  COMPILE := loader-enh202-v1.bin loader-enh202-v1.uImage
+  COMPILE/loader-enh202-v1.bin := loader-okli-compile
+  COMPILE/loader-enh202-v1.uImage := append-loader-okli enh202-v1 | \
+	pad-to $$$$(BLOCKSIZE) | lzma | uImage lzma
+  IMAGES += factory.bin
+  IMAGE/factory.bin := append-squashfs-fakeroot-be | pad-to $$$$(BLOCKSIZE) | append-kernel | \
+	pad-to $$$$(BLOCKSIZE) | append-rootfs | pad-rootfs | check-size | pad-to 4928k | \
```

Comment:
Why do we have pad-to 4928k here, but nothing comparable for the ens202ext. Just tried to make a common device definition when I found this.

Reply:
Originally I thought it was helping rootfs_data to be identified and erased, but actually I had incorrect partitions (I was experimenting with 4k size / 4k offset start and end partitions). I think the before-upgrade.sh is enough to make sure firstboot is clean but I'll make sure.